R/chartutils.R

```diff
@@ -863,17 +863,18 @@ setSubtitle <- function(subtitle, subtitle.font, margins)
                 yref = "paper", y = 1.0, yanchor = "bottom", showarrow = FALSE))
 }
 
-setFooter <- function(footer, footer.font, margins)
+setFooter <- function(footer, footer.font, margins, x.align = "center")
```

Comment:
Are the elements of the output object documented somewhere? Are they parameters used by plotly? If so, can you add a comment pointing to where they are defined?

Reply:
I've added a comment with a link to the plotly documentation just before setTittle (because the documentation applies to setTItle, setSubtitle and setFooter).